### PR TITLE
clock_1_1

### PR DESCRIPTION
**Pull request by SergeyDeryabin19**

### CODE REVIEW

File: clock.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    
+}
+
+
+
+def screen_clear():
+    os.system("CLS")
+    
+
+def print_result(continue_hour_1, continue_hour_2, continue_minute_1,\
```
Comment (dvk-net) [inline clock.py:102]: I would insert all the digits inside a single structure (list, tuple...)

File: clock.py [hunk 2]
```diff
@@ -0,0 +1,163 @@
+import datetime
+import os
+import time
+
+
+
+dict_num=\
```
Comment (dvk-net) [inline clock.py:7]: I would pre-split the digits into raws

[
"xxxx",
"x    x"
]

**Review by dvk-net — APPROVED** [review]
very good

File: clock.py (hunk 2 above)
Comment (SergeyDeryabin19) [reply clock.py:7]: ok, inderstand that

**Pull request merged**